README.md

```diff
@@ -74,6 +76,23 @@ jobs:
           Get-AzVM -ResourceGroupName "ActionsDemo"
 ```
 
+## Sample to connect to Azure US Government cloud
+
+```
+   - name: Login to Azure US Gov Cloud with CLI 
+     uses: azure/login@v1.2
+        with:
+          creds: ${{ secrets.AZURE_US_GOV_CREDENTIALS }}
```

Comment:
Please use the version V1 only... coz we would be moving the V1 tag to point to the latest minor version tag which is v1.2.